runtime/vm/module.xml

```diff
@@ -90,6 +90,12 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<makefilestub data="UMA_OBJECTS:=$(filter-out zcinterp%,$(UMA_OBJECTS))\n">
 				<exclude-if condition="spec.flags.arch_s390"/>
 			</makefilestub>
+			<makefilestub data="UMA_OBJECTS:=$(filter-out rv64cinterp%,$(UMA_OBJECTS))\n">
+				<exclude-if condition="spec.flags.arch_riscv and spec.flags.J9VM_ENV_DATA64"/>
+			</makefilestub>
+			<makefilestub data="UMA_OBJECTS:=$(filter-out rv64cinterpnojit%,$(UMA_OBJECTS))\n">
```

Comment:
I don't think the `\n` is necessary nor desirable in the stubs.

Reply:
Fixed as suggested above.